airflow/providers/google/common/hooks/base_google.py

```diff
@@ -207,13 +207,20 @@ def _get_access_token(self) -> str:
         """
         return self._get_credentials().token
 
+    @staticmethod
+    def _build_http() -> httplib2.Http:
```

Comment:
We do not need to give the opportunity to override this method. If the user needs it, the user can overwrite authorize.  It would make sense if _http were used in several places.